PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/DirectoryAdapter.java

```diff
@@ -110,13 +112,24 @@ public void onClick(View v) {
                 SharedPreferences.Editor editedPreferences = sharedPref.edit();
                 ToggleButton star = (ToggleButton) v;
                 boolean starred = star.isChecked();
-                String currentName = currentPerson.getName();
+                String currentName = currentPerson.name;
                 if (starred) {
                     if (currentName != null) {
                         starredContacts.add(currentName);
+                        StringBuilder stringBuilder = new StringBuilder();
+                        stringBuilder.append(currentPerson.affiliation);
+                        stringBuilder.append(mContext.getString(R.string.directory_delim));
```

Comment:
Just `String delim = mContext.getString(R.string.directory_delim)` the first time and use it everywhere, more clear that way.